pennylane/io.py

```diff
@@ -407,6 +407,65 @@ def from_qiskit_op(qiskit_op, params=None, wires=None):
         raise RuntimeError(_MISSING_QISKIT_PLUGIN_MESSAGE) from e
 
 
+def from_qiskit_noise(noise_model, **kwargs):
+    """Converts a Qiskit `NoiseModel <https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.noise.NoiseModel.html>`__
+    into a PennyLane :class:`~.NoiseModel`.
+
+    Args:
+        noise_model (qiskit_aer.noise.NoiseModel): a Qiskit ``NoiseModel`` instance.
+        kwargs: Optional keyword arguments for conversion of the noise model.
+
+    Keyword Arguments:
```

Comment:
My last comment would be to ask why we have `**kwargs` rather than just listing `verbose` and `decimal_places` explicitly.

Reply:
This was just to remain in parity with the `load_quantum_noise` function in the pl-qiskit plugin. These could be shifted to the `args` as well if you'd like.

Reply:
Was there a motivation for the `load_quantum_noise` function in the pl-qiskit plugin to use `**kwargs`? If not, I'd prefer to switch to explicit args. Thanks @obliviateandsurrender!

Reply:
Any preference @isaacdevlugt?

Reply:
I think the arguments should be explicit 🙂